pipeline/lib/pipeline.ml

```diff
@@ -193,15 +193,16 @@ let process_pipeline ~(docker_config : Docker_config.t) ~conninfo
         |> let> api, repo = repo in
            Github.Api.refs api repo
       in
+      let default_branch = Github.Api.default_ref refs in
+      let ref_map = Github.Api.all_refs refs in
       Github.Api.Ref_map.fold
         (fun key head _ ->
           let head = `Github head in
           match key with
-          | `Ref "refs/heads/master" -> pipeline ~head ~branch:"master" ()
+          | `Ref branch -> if branch = default_branch then pipeline ~head ~branch () else Current.return ()
```

Comment:
This will capture `refs/heads/master` as `branch` instead of using `"master"` as in the current version.

I think we should keep the branch name the same, i.e., continue using `"master"`.

Ideally this should apply to the default branch too.